.devcontainer/website/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Node.js & TypeScript",
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:16-bullseye",
+  "workspaceMount": "source=${localWorkspaceFolder},target=/workspaces/onefetch,type=bind,consistency=cached",
+  "workspaceFolder": "/workspaces/onefetch/docs/vercel",
+  "extensions": ["esbenp.prettier-vscode", "svelte.svelte-vscode"],
+  "postCreateCommand": "npm ci",
+  "postAttachCommand": "npm run dev -- --port 3000",
```

Comment:
why not merge the two into : `"postCreateCommand": "npm ci && npm run dev -- --port 3000"`?

Reply:
Post create only happens once, when first creating the codespace, while post attach runs each time you connect to the codespace. This way you can stop the codespace and have the dev server run again next time you connect without needing to rebuild the container.